CppCoreGuidelines.md

```diff
@@ -18547,15 +18547,19 @@ You can call C from C++:
 
 ##### Example
 
-You can call C++ from C:
+You can call C++ from C, if needed do not forget to convert the error management for C compilers may expect `noexcept` functions:
```

Comment:
/, if needed do/ if needed.\n Do/

/management for C/handling, as C/

Reply:
Even with those edits it's unclear.